apps/trace-import/src/TraceImportCommandLine.ts

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import colors from 'colors';
+import {
+  CommandLineParser,
+  CommandLineFlagParameter,
+  CommandLineStringParameter,
+  CommandLineChoiceParameter
+} from '@rushstack/ts-command-line';
+import { InternalError } from '@rushstack/node-core-library';
+
+import { ResolutionType, traceImport } from './traceImport';
+
+export class TraceImportCommandLine extends CommandLineParser {
```

Comment:
```suggestion
export class TraceImportCommandLineParser extends CommandLineParser {
```

Following convention in the other projects.

Reply:
Fixed